src/psyclone/tests/domain/lfric/lfric_builtins_test.py

```diff
@@ -4312,7 +4596,7 @@ def test_sign_X_by_value(tmpdir, monkeypatch, annexed, dist_mem):
 
 
 def test_multiple_builtin_set(tmpdir, monkeypatch, annexed, dist_mem):
-    ''' Tests that we generate correct code when we have an invoke
+    '''Tests that we generate correct code when we have an invoke
```

Comment:
`'invoke'`?

Reply:
Oops, that appears to be the only place in the file within a docstring where I've not done that.